### ISSUE TITLE
Assess and improve accessibility of documentation

### ISSUE THREAD

**Issue by MarionBWeinzierl** [N8-CIR-Bede/documentation#42]
Liaise with members of the WHPC action group and discuss how to assess the accessibility of the Bede documentation, and how to improve it.

**Comment by MarionBWeinzierl** [comment]
See, for example: https://webaccess.berkeley.edu/resources/tips/web-accessibility

**Comment by Fehings** [comment]
Following on from the WHPC action group meeting: 
- look into automated tools to assess the current accessibility of these docs
- Niki to create a guidelines memo for the N8 website and Bede documentation re: accessibility best practice
- generate github issues to fix any current accessibility problems/ where we aren't meeting the guidelines
I will raise this at the task force meeting as well.

**Comment by kilicomu** [comment]
I put the docs through WAVE; you can see the report here:

https://wave.webaim.org/report#/https://bede-documentation.readthedocs.io/en/latest/

Definitely worth putting the site through a few more accessibility assessment tools to capture different perspectives, but this will probably be a decent start.

**Comment by ptheywood** [comment]
`pa11y` is a node application which evaluates accessibility of html pages relative to WCAG guidelines. It should be relatively simple to integrate this into the website CI. 

If issues are raised within the sphinx generated code html output it may be non-trivial to address all the highlighted challenges however, so we may just wish to add annotations to the CI runs (and encourage local testing) rather than fail builds. 

It looks like it should also be possible to confiure pa11y to skip certain rules if there are certain things we cannot address without modifying sphinx / the sphinx theme.

https://github.com/pa11y/pa11y

**Comment by ptheywood** [comment]
Closing since #129 has been merged. This could be enforced via CI in the future
